value/src/main/java/com/google/auto/value/processor/BuilderSpec.java

```diff
@@ -131,10 +132,10 @@
     ImmutableSet<ExecutableElement> toBuilderMethods(
         Types typeUtils, Set<ExecutableElement> abstractMethods) {
 
-      ImmutableList<String> builderTypeParamNames =
-          FluentIterable.from(builderTypeElement.getTypeParameters())
-              .transform(SimpleNameFunction.INSTANCE)
-              .toList();
+      List<String> builderTypeParamNames =
+          builderTypeElement.getTypeParameters().stream()
+              .map(e -> e.getSimpleName().toString())
+              .collect(toList());
```

Comment:
There is an `ImmutableList.Builder` usage below (lines 145-150) that I believe could be replaced with streams as well (if one wants to go all out. :wink:)